<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomePage</class>
 <widget class="QWidget" name="HomePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QTabWidget" name="Homepge">
     <property name="styleSheet">
      <string notr="true">Homepage{&quot;QTabWidget::pane{ border:none; }&quot;
                &quot;QTabWidget::tab-bar{ alignment:left; }&quot;
                &quot;QTabBar::tab{ background:transparent; color:white; min-width:30ex; min-height:10ex; }&quot;
                &quot;QTabBar::tab:hover{ background:rgb(255, 255, 255, 100); }&quot;
                &quot;QTabBar::tab:selected{ border-color: white; background:white; color:green; }&quot;}</string>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::South</enum>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="HOME">
      <attribute name="title">
       <string>Home</string>
      </attribute>
      <widget class="QFrame" name="frame">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>50</y>
         <width>681</width>
         <height>141</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="booking">
          <property name="styleSheet">
           <string notr="true">border-radius: 50px;
    background: qradialgradient(cx:0.5, cy:0.5, radius: 1, fx:0.5, fy:0.5, stop:0 #0099FF, stop:1 #0044CC);
    color: white;
    font-family: &quot;楷体&quot;;</string>
          </property>
          <property name="text">
           <string>预约挂号</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="prescription_inquiry">
          <property name="styleSheet">
           <string notr="true">border-radius: 50px;
    background: qradialgradient(cx:0.5, cy:0.5, radius: 1, fx:0.5, fy:0.5, stop:0 #0099FF, stop:1 #0044CC);
    color: white;
    font-family: &quot;楷体&quot;;</string>
          </property>
          <property name="text">
           <string>处方查询</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="medica_record">
          <property name="styleSheet">
           <string notr="true">border-radius: 50px;
    background: qradialgradient(cx:0.5, cy:0.5, radius: 1, fx:0.5, fy:0.5, stop:0 #0099FF, stop:1 #0044CC);
    color: white;
    font-family: &quot;楷体&quot;;</string>
          </property>
          <property name="text">
           <string>电子病例</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="health_evaluate">
          <property name="styleSheet">
           <string notr="true">border-radius: 50px;
    background: qradialgradient(cx:0.5, cy:0.5, radius: 1, fx:0.5, fy:0.5, stop:0 #0099FF, stop:1 #0044CC);
    color: white;
    font-family: &quot;楷体&quot;;</string>
          </property>
          <property name="text">
           <string>健康评估</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QFrame" name="frame_5">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>230</y>
         <width>681</width>
         <height>151</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">frame{url(:/resource111/1.png);}</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="Message">
      <attribute name="title">
       <string>Message</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>在线问诊·消息栏</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="updatePushbutton">
         <property name="text">
          <string>update</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOn</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOn</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>712</width>
            <height>355</height>
           </rect>
          </property>
          <layout class="QFormLayout" name="formLayout"/>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="My">
      <attribute name="title">
       <string>My</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>个人信息修改</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QFrame" name="frame_4">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_17">
               <property name="text">
                <string>就诊号：１２３４５６７</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,100,1,100,1">
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QFrame" name="frame_3">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <widget class="QLabel" name="label_12">
                 <property name="text">
                  <string>电话号码</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="boxTelephone"/>
               </item>
               <item>
                <widget class="QLabel" name="label_13">
                 <property name="text">
                  <string>地址</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="boxAddress"/>
               </item>
               <item>
                <widget class="QLabel" name="label_14">
                 <property name="text">
                  <string>电子邮件</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="boxEmail"/>
               </item>
               <item>
                <widget class="QLabel" name="label_15">
                 <property name="text">
                  <string>修改密码：</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_13"/>
               </item>
               <item>
                <widget class="QLabel" name="label_16">
                 <property name="text">
                  <string>确认密码：</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_14"/>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QFrame" name="frame_2">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>姓名：</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="boxName"/>
               </item>
               <item>
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string>年龄：</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="boxAge"/>
               </item>
               <item>
                <widget class="QLabel" name="label_9">
                 <property name="text">
                  <string>身份证号：</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="boxID"/>
               </item>
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_7">
                 <property name="styleSheet">
                  <string notr="true">border-radius: 10px;
background-color: #F0F0F0;
                        border: 2px solid #C0C0C0;
                        padding: 5px;
                        
</string>
                 </property>
                 <property name="text">
                  <string>修改资料</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_8">
                 <property name="styleSheet">
                  <string notr="true">border-radius: 10px;
background-color: #F0F0F0;
                        border: 2px solid #C0C0C0;
                        padding: 5px;
                        
</string>
                 </property>
                 <property name="text">
                  <string>提交</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
